client/state/global-styles/actions.ts

```diff
@@ -0,0 +1,55 @@
+import wpcom from 'calypso/lib/wp';
+import type { ActiveTheme, GlobalStyles } from '@automattic/data-stores';
+
+export function getGlobalStylesId( siteSlug: string, stylesheet: string ) {
```

Comment:
Seems that we can either pass `siteId` or `siteSlug` here. Can we support either?